Presentation.Web/app/helpers/RenderFieldsHelper.ts

```diff
@@ -65,7 +65,8 @@
         }
 
         static renderDate(date: Date | undefined | null) {
-            if (!!date) {
+            if (!!date ||
```

Comment:
I think it should be &&, not || here because !! means that is is actually defined and not null or empty string, and only then it will be allowed to check the format right?